### PR TITLE
Update docs: mongo replicaset script moved to new repo

### PR DESCRIPTION
**Pull request by taliaga**
https://github.com/smartsdk/smartsdk-recipes/issues/2

### CODE REVIEW

File: recipes/utils/mongo-replicaset/readme.md [hunk 1]
```diff
@@ -117,17 +117,15 @@ If a node goes down, the replica-set will be automatically reconfigured at the a
 
 _NOTE_: If you don't want to have a replica in every node of the swarm, the solution for now is using a combination of constraints and node tags. You can read more about this in [this Github issue](https://github.com/docker/docker/issues/26259).
 
-For further details, refer to the *[docker-compose.yml](https://github.com/smartsdk/smartsdk-recipes/blob/master/recipes/utils/mongodb/replica/docker-compose.yml)* file or *[replica_ctrl.py](https://github.com/smartsdk/smartsdk-recipes/blob/master/recipes/utils/mongodb/replica/scripts/replica_ctrl.py)*.
+For further details, refer to the [mongo-rs-controller-swarm](https://github.com/smartsdk/mongo-rs-controller-swarm) repository, in particular the *[docker-compose.yml](hhttps://github.com/smartsdk/mongo-rs-controller-swarm/blob/master/docker-compose.yml)* file or the *[replica_ctrl.py](https://github.com/smartsdk/mongo-rs-controller-swarm/blob/master/src/replica_ctrl.py)* controller script.
```
Comment (chicco785) [inline recipes/utils/mongo-replicaset/readme.md]: hhttps -> https

File: recipes/utils/mongo-replicaset/readme.md [hunk 2]
```diff
@@ -117,17 +117,15 @@ If a node goes down, the replica-set will be automatically reconfigured at the a
...
+So, the first approach is to find out this information from the docker api. Also, since the recipe is expected to be self-contained and work without dependencies on things running outside the swarm, we need to get this information from a container running in the swarm. My understanding is that such an introspective api to safely retrieve this kind of information is yet to come (e.g [this issue](https://github.com/docker/docker/issues/8427) and related ones such as [this one](https://github.com/docker/docker/issues/1143#issuecomment-233152700)). So for now this is depending on access to the host's docker socket **(terribly insecure workaround)**. A different approach to explore would be passing to the script at runtime the list of IPs. IPs of the containers if possible, or if not, IPs of the nodes where the services are deployed and use different ports in each replica member so as to avoid clashes in the docker ingress network. Or, something considering a custom IPM via plugins.
```
Comment (chicco785) [inline recipes/utils/mongo-replicaset/readme.md]: i don't think this discussion make sense in this repo

File: recipes/utils/mongo-replicaset/readme.md (hunk 1 above)
Comment (taliaga) [reply recipes/utils/mongo-replicaset/readme.md]: nice catch.

File: recipes/utils/mongo-replicaset/readme.md (hunk 2 above)
Comment (taliaga) [reply recipes/utils/mongo-replicaset/readme.md]: Agree. Removed.

**Review by chicco785 — APPROVED** [review]

**Pull request merged**